forge/test/PoolFactory/DeployPool.t.sol

```diff
@@ -0,0 +1,330 @@
+//SPDX-License-Identifier: CC-BY-NC-ND-4.0
+pragma solidity 0.8.7;
+
+import "../helpers/PoolSetupHelper.t.sol";
+import "interfaces/ILeveragedPool.sol";
+import "testutils/TestClones.sol";
+import "implementation/SMAOracle.sol";
+
+contract PoolFactoryDeployPoolTest is PoolSetupHelper {
+    address constant NON_DAO = ADDR_4;
+
+    uint32 internal frontRunningInterval = 20;
+    uint32 internal updateInterval = 100;
+    uint256 internal fee = 0.1 ether;
+    uint16 internal leverage = 1;
+
+    function setUp() public {
+        setupPoolAndTokenContracts(
+            POOL_CODE,
+            frontRunningInterval,
+            updateInterval,
+            leverage,
+            FEE_RECEIVER,
+            fee,
+            0,
+            0,
+            0
+        );
+    }
+
+    function test_cannot_deploy_when_not_called_by_oracleWrapper_owner_and_with_valid_params() public {
```

Comment:
Is this the common naming convention? Not trying to say it's bad but more just curious to make sure we're following convention.

Reply:
Not a prescription by any means, but that's the verbiage used in the foundry book. https://book.getfoundry.sh/forge/writing-tests.html

The only stylistic choice I made was using snake instead of camel case.

Reply:
Yep looks good.